src/app/[locale]/(profile)/types/user.type.ts

```
@@ -0,0 +1,20 @@
+// src/types/user.ts
+export type User = {
+  _id: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  password?: string | null;
+  gender?: string | null;
+  phone?: string | null;
+  photo?: string | null;
+  role?: string | null;
+  wishlist?: any[];  
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)